service/src/main/kotlin/fi/espoo/evaka/assistanceneed/preschooldecision/AssistanceNeedPreschoolDecisionController.kt

```diff
@@ -333,6 +325,7 @@ class AssistanceNeedPreschoolDecisionController(
             .also { Audit.ChildAssistanceNeedPreschoolDecisionDelete.log(targetId = id) }
     }
 
+    // TODO: Unused endpoint?
```

Comment:
copy paste from old implementation but didn't find any frontend using this, not sure if I missed something or can this be removed